src/current/v23.2/physical-cluster-replication-overview.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Physical Cluster Replication
+summary: An overview of CockroachDB's physical cluster replication.
+toc: true
+docs_area: manage
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v23.2" %} CockroachDB physical cluster replication continuously sends all data at the byte level from a _primary_ cluster to an independent _standby_ cluster. Existing data and ongoing changes on the active primary cluster, which is serving application data, replicate asynchronously to the passive standby cluster.
+
+In a disaster recovery scenario, you can [_cut over_]({% link {{ page.version.version }}/cutover-replication.md %}) from the unavailable primary cluster to the standby cluster. This will stop the replication stream, reset the standby cluster to a point in time where all ingested data is consistent, and mark the standby as ready to accept application traffic.
+
+{% comment %}A lot of links at the end of this section to general CV docs{% endcomment %}
+
+## Use cases
+
+You can use physical cluster replication in a disaster recovery plan to:
+
+- Meet your RTO (Recovery Time Objective) and RPO (Recovery Point Objective) requirements. Physical cluster replication provides lower RTO and RPO than [backup and restore]({% link {{ page.version.version }}/backup-and-restore-overview.md %}). {% comment %}link here to upcoming DR tool comparative table{% endcomment %}
+- Automatically replicate everything in your primary cluster to recover quickly from a control plane or full cluster failure.
+- Protect against region failure when you cannot use individual [multi-region clusters]({% link {{ page.version.version }}/multiregion-overview.md %}). For example, if you have a two-datacenter architecture and do not have access to three regions. Or, you need low-write latency in a single region. Physical cluster replication allows for an active-passive (primary-standby) structure across two clusters with the passive cluster in a different region.
```

Comment:
The "for example" and the following sentence are fragments.

Reply:
I have adjusted these, hopefully they are little clearer now and expand on the original clause.